lib/entities/default_settings_migration.dart

```diff
@@ -843,7 +843,7 @@ Future<void> changeDefaultMoneroNode(
     }
   });
 
-  final newCakeWalletNode = Node(uri: newCakeWalletMoneroUri, type: WalletType.monero);
+  final newCakeWalletNode = Node(uri: newCakeWalletMoneroUri, type: WalletType.monero, trusted: true);
```

Comment:
like this it will be set to trusted for new installs only, can you please add it as a migration step (i.e case 42:) and change it for existing users as well

Reply:
also let's add to the restore from QR card the Cupcake option
i.e just change the description of the `Scan QR code` card in the restore options to be something like

> Add a cold wallet or recover a paper wallet or add your read-only wallet from Cupcake

Reply:
@tuxpizza if you have another idea to show cupcake here please chime in